### ISSUE TITLE
replication job exit with error failed to get job tracker

### ISSUE THREAD

**Issue by ZPerling** [goharbor/harbor#11619]
Harbor version v1.10.1-f3e11715, deploy use docker-compose

Some replication job failed，error log：
```
Apr  8 16:04:55 172.21.0.1 jobservice[1982]: 2020-04-08T08:04:55Z [INFO] [/jobservice/worker/cworker/c_worker.go:74]: Job incoming: {"name":"REPLICATION","id":"c29e45c967b315a2e36970d8","t":1586333095,"args":null}
Apr  8 16:04:55 172.21.0.1 jobservice[1982]: 2020-04-08T08:04:55Z [ERROR] [/jobservice/runner/redis.go:67]: Job 'REPLICATION:c29e45c967b315a2e36970d8' exit with error: failed to get job tracker: {"code":10010,"message":"object is not found","details":"c29e45c967b315a2e36970d8"}
```

what is happend?

**Comment by steven-zou** [comment]
@ZPerling 

Job data in Redis might be lost or expired.

**Comment by steven-zou** [comment]
@ZPerling 

 Improvements have been done to the tacker getting code, code has been merged into 2.0 and master. So you can upgrade to the latest 1.10.X or 2.0.

https://github.com/goharbor/harbor/pull/11748

**Comment by ZPerling** [comment]
@steven-zou thks
